### PR TITLE
🚀 Add Monix integration

### LINKED ISSUE #57: Add Monix integration

### PR DESCRIPTION
**Pull request by tkrs**
Closes #57

---
This change is [Reviewable](https://reviewable.io/reviews/yanana/agni/58)

### CODE REVIEW

File: monix/src/main/scala/agni/monix/Task.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.concurrent.Executor
+
+import agni.cache.CachedPreparedStatementWithGuava
+import _root_.cats.MonadError
+import com.datastax.driver.core.{ PreparedStatement, ResultSet, Statement }
+import com.google.common.cache.Cache
+import com.google.common.util.concurrent.{ FutureCallback, Futures }
+import _root_.monix.cats._
+import _root_.monix.execution._
+import _root_.monix.eval.{ Coeval, Task => MTask }
```
Comment (yanana) [inline monix/src/main/scala/agni/monix/Task.scala:13]: Why don't you use simply `Task`?

File: monix/src/main/scala/agni/monix/package.scala [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package agni
+
+import cats.syntax.either._
+import cats.{ Eval, MonadError }
+import _root_.monix.eval.{ Task => MTask }
+
+package object monix {
+
+  object cats {
+    implicit val monixTaskToCatsMonadError: MonadError[MTask, Throwable] =
```
Comment (yanana) [inline monix/src/main/scala/agni/monix/package.scala]: How about simply `taskToMonadError`?

Comment (tkrs) [reply monix/src/main/scala/agni/monix/package.scala]: Thanks! I will do it.

File: monix/src/main/scala/agni/monix/Task.scala (hunk 1 above)
Comment (tkrs) [reply monix/src/main/scala/agni/monix/Task.scala:13]: In this file, use the `Task` for the class name.

**Review by yanana — APPROVED** [review]
LGTM

**Pull request merged**